cypher-shell/src/main/java/org/neo4j/shell/state/BoltStateHandler.java

```diff
@@ -133,7 +133,20 @@ private void silentDisconnect() {
         }
     }
 
+    /**
+     * Reset the current session. This rolls back any open transactions.
+     */
     public void reset() {
-//        session.reset();
+        // TODO once drivers release next milestone
+        //if (session != null) {
```

Comment:
So we can do away with null checks but check whether the cypher shell is connected instead. I just really don't like null checks :(